content/beginner/110_irsa/deploy.md

```diff
@@ -62,6 +62,49 @@ Output example
 2020-02-12 01:48:05 eksworkshop-logs
 {{< /output >}}
 
+{{% notice info %}}
+If you have an output, please move on to "List EC2 Instances."
+
+If the output is empty, it is possible your account doesn't have any s3 buckets.
+Please try to run theses extra commands.
+{{% /notice %}}
+
+Let's create an S3 bucket.
+
+```bash
+aws s3 mb s3://eksworkshop-$ACCOUNT_ID-$AWS_REGION --region $AWS_REGION
```

Comment:
Can we make sure to add this to the cleanup at the end of the chapter?

Reply:
Good call!  Here it is!